elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/dsl/CompletionDsl.scala

```diff
@@ -20,35 +20,57 @@ package com.sumologic.elasticsearch.restlastic.dsl
 
 trait CompletionDsl extends DslCommons with QueryDsl {
 
-  case class Suggest(text: String, completion: Completion) extends Query with RootObject {
+  case class SuggestRoot(textOpt: Option[String], suggestions: List[Suggestion]) extends Query with RootObject {
     val _suggest = "suggest"
     val _text = "text"
+
+
+    override def toJson: Map[String, Any] = {
+      val globalTextJson = if (textOpt.isDefined) Map(_text -> textOpt.get) else Map.empty
```

Comment:
We can probably do 
`val globalTextJson = Map.empty ++ textOpt.map(text => _text -> text)`